### DISCUSSION TITLE
Lookup command returning too many opposite faction bots.

### DISCUSSION THREAD

**Discussion by jdubs187** [trickerer/Trinity-Bots#387] [Classes]
Is there a way to alter the created bots by the `.npcbot lookup` command?? 

I"m specifically a horde player and over half of the bot ids returned via the lookup command are alliance races.
Some races aren't created at all for some classes. 
It won't create a Tauren DK for example.

edit: AzerothCore rev. b938c1d427f5

**Comment by jdubs187** [comment]
So...I've found all the entries in the `creature_template` table.
and the corresponding `class` and `race` entries for each id located in `creature_temlate_npcbot_extras`
Can I just do a batch change on these numbers.
Say alter all draenei entries to tauren entries instead?

I don't mind doing it manually at all.
I'll wait to hear from you before doing this because I don't want to cause issues in the db.
I'm completely new to sql.

**Reply by trickerer** [reply]
All pre-existing bots in `creature_template` created based on class trainer npcs, that's why their numbers are so limited for some classes.

Altering `creature_template_npcbot_extras` is ill-advised since it's not enough and will only make bot models look ugly.
Your best bet is to create new bots you need using `.npcbot createnew` command. Check README on how to use it.

**Reply by jdubs187** [reply]
ah ty.
I shoulda been more thorough reading the manual.
I'll give the createnew command a  try. 

I appreciate the reply.

**Comment by trickerer** [comment]
https://github.com/trickerer/Trinity-Bots/commit/e67a5ea35d58d57721af85105bc5943cf9f73710
